tests/editing-tools.test.ts

```diff
@@ -3,6 +3,7 @@ import { binaryOnPathTest } from "./index.ts";
 const textManipTools: { binary: string }[] = [
```

Comment:
Should the identifier also be changed to either match the new file name or be more general?